src/components/navBar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+const navBar = () => (
+  <div className="navBar">
+    <h1>Book Store</h1>
+    <Router>
+      <nav>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/categories">Categories</Link>
+          </li>
+          <li>
+            <Link to="/profile">Profile</Link>
```

Comment:
- [ ] This profile link on the navbar is unnecessary as there is no component related to it. Kindly remove it as it is unneeded.